nvflare/apis/storage.py

```diff
@@ -62,14 +65,14 @@ def update_meta(self, uri: str, meta: dict, replace: bool):
         Returns:
 
         Raises exception when:
-
-        - no such object
         - invalid args
+        - no such object
         - error updating the object
```

Comment:
I think in the spec's docstring we don't need to specify what exception needs to be raised.

As we don't expect that from spec, that should come from the implementation side.

Reply:
On the other hand, if we do require users to raise these exception in these situations, then they can stay in the docstring.

Reply:
Agreed, I will add try-excepts around the minio API calls to throw specified exceptions as well